src/App.js

```diff
@@ -1,32 +1,30 @@
 import React from 'react'
 import './App.css'
-import Button from './atoms/Button'
-import Heading from './atoms/Heading'
-import Text from './atoms/Text'
 import { createGlobalStyle } from 'styled-components'
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
+import Artist from './pages/Artist'
 
 const GlobalStyle = createGlobalStyle`
   body {
     font-size: 14px;
+    background-color: #222;
+    color: ${({ theme }) => theme.colors.white};
+  }
+
+  a {
+    text-decoration: none;
   }
 `
 
 function App() {
   return (
     <div className="App">
```

Comment:
Do you still use these styles from `App.css`? If not, remove the `App.css` file and this class name.